Python/ceval.c

```diff
@@ -1077,6 +1078,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 // Jump here from _EXIT_TRACE
 exit_trace:
     _PyFrame_SetStackPointer(frame, stack_pointer);
+    frame->instr_ptr = next_uop[-1].target + _PyCode_CODE((PyCodeObject *)frame->f_executable);
```

Comment:
Maybe assert that target is not 0?

Reply:
Sometimes exit the trace immediately upon making a call, if it appears to be recursive.
So, target can be zero.